opentuner/search/technique.py

```diff
@@ -204,15 +223,20 @@ def call_main_generator(self):
                                     self.pending_tests)
         if self.pending_tests:
           self.rounds_since_novel_request = 0
-          yield None # wait
+          yield False # wait
 
 #list of all techniques
 the_registry = list()
 
+#list of composable techniques
+the_composable_registry = list()
 
 def register(t):
   the_registry.append(t)
 
+def register_composable(t):
```

Comment:
Duplicating all this registry code is messy.  Lets keep just a single `register()` / `the_registry` and use `isinstance(ComposableEvolutionaryTechnique ,...)` to determine which techniques are subclasses of composable